src/zenml/new/pipelines/pipeline.py

```diff
@@ -677,6 +678,13 @@ def _run(
                 deployment=deployment, local_repo_context=local_repo_context
             )
 
+            if prevent_build_reuse:
+                logger.warning(
+                    "Passing `prevent_build_reuse=True` to "
+                    "`pipeline.with_opitions(...)` is deprecated. Use "
+                    "`DockerSettings.prevent_build_reuse` instead."
+                )
```

Comment:
Can this setting be passed up to the pipeline in such a case, so we can keep it backward compatible to the max?

Reply:
You mean directly on the `@pipeline` decorator?

For now this is fully backwards compatible (`pipeline.with_options(prevent_build_reuse=True` still works) in any case I think

Reply:
Ah, nice, all good as long as it is backward-compatible - I have misread this, it seems